tests/_utils/model.js

```diff
@@ -9,60 +9,143 @@ import TreeWalker from '/ckeditor5/engine/treemodel/treewalker.js';
 import Range from '/ckeditor5/engine/treemodel/range.js';
 import Position from '/ckeditor5/engine/treemodel/position.js';
 import Text from '/ckeditor5/engine/treemodel/text.js';
+import RootElement from '/ckeditor5/engine/treemodel/rootelement.js';
 import Element from '/ckeditor5/engine/treemodel/element.js';
+import DocumentFragment from '/ckeditor5/engine/treemodel/documentfragment.js';
+import Selection from '/ckeditor5/engine/treemodel/selection.js';
+import Document from '/ckeditor5/engine/treemodel/document.js';
 
 /**
- * Writes the contents of the document to an HTML-like string.
+ * Writes the contents of the {@link engine.treeModel.Document Document} to an HTML-like string.
  *
  * @param {engine.treeModel.Document} document
- * @param {String} rootName
  * @param {Object} [options]
- * @param {Boolean} [options.selection] Whether to write the selection.
+ * @param {Boolean} [options.withSelection] Whether to write the selection.
+ * @param {Boolean} [options.rootName='main'] Name of the root from which data should be stringified. If not provided
+ * default `main` name will be used.
  * @returns {String} The stringified data.
  */
-export function getData( document, rootName, options ) {
+export function getData( document, options = {} ) {
+	const withSelection = !!options.withSelection;
```

Comment:
I prefer make `withSelection` true by default. We could rename it to `withoutSelection`, since it is more natural to have options to be false by default.